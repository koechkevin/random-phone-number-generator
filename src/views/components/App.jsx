import React from 'react';
import { Provider } from 'react-redux';
import { Router } from 'react-router-dom';
import { createBrowserHistory } from 'history';
import store from '../redux/store';
import Home from './Home';
import NavBar from './NavBar';
import '../styles/styles.scss';

const history = createBrowserHistory();

const App = () => (
  <Provider store={store}>
    <Router history={history}>
      <NavBar />
      <Home />
    </Router>
  </Provider>
);
export default App;
