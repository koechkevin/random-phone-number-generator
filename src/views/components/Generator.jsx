import React, { Component } from 'react';

class Generator extends Component {
  render() {
    return (
      <div className="gen">
        <form onSubmit={(e) => { e.preventDefault(); }}>
          Enter the number of phone numbers to generate
          <br/>
          <input type="number"/>
          <input type="submit" value="Generate"/>
        </form>
      </div>
    );
  }
}

export default Generator;
