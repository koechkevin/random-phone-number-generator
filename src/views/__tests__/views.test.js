import { mount } from 'enzyme/build';
import React from 'react';
import App from '../components/App';
import Generator from '../components/Generator';

describe('test', () => {
  it('passes', () => {
    expect(mount(<App />)).toMatchSnapshot();
    expect(mount(
      <Generator/>,
    )).toMatchSnapshot();
  });
});
