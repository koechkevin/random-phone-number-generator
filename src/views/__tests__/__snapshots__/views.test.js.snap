// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test passes 1`] = `ReactWrapper {}`;

exports[`test passes 2`] = `ReactWrapper {}`;
